templates/highlighterCss.txt

```diff
@@ -0,0 +1,11 @@
+.popcode_HIGHLIGHTER {
```

Comment:
Wanted to put this file in src/previewFrame but the import wouldn't work

Reply:
I think that’s basically correct—webpack is configured to treat everything in here as a raw text file, whereas it would assume files elsewhere are JavaScript. I would probably name this `highlighter.css` though?

Reply:
Okay thanks! Changed to highlighter.css